draft-ietf-rats-corim.md

```diff
@@ -1000,6 +1000,41 @@ measurements for the Target Environment.
 
 [^issue] https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/10
 
+# CoBOM {#sec-cobom}
+
+A BOM object represents the signal for the verifier to activate the listed
```

Comment:
Should expand the BOM acronym before using the acronym.
Also, wording should be more clear that the `tags-list` in the `concise-bom-tag` behaves like a signaling mechanism from a product vendor to a Verifier that _activates_ the tags in `tags-list` for use in an Evidence appraisal process.

Reply:
done in a5638b2